client/my-sites/term-tree-selector/add-term.jsx

```diff
@@ -220,7 +220,7 @@ export default connect(
 	( state, ownProps ) => {
 		const { taxonomy, postType } = ownProps;
 		const siteId = getSelectedSiteId( state );
-		const { labels } = getPostTypeTaxonomy( state, siteId, postType, taxonomy ) || {};
+		const { labels } = getPostTypeTaxonomy( state, siteId, postType, taxonomy ) || { labels: {} };
```

Comment:
Maybe our good friend `_.get` can help us make this clearer? 😄 

```js
const labels = get( getPostTypeTaxonomy( state, siteId, postType, taxonomy ), 'labels', {} );
```

I wonder if it makes sense at a certain point to create a selector for getting a taxonomy's labels, not only to avoid these accessor issues, but also to provide many of the [hard-coded fallback labels](https://github.com/Automattic/wp-calypso/blob/b109436609d955f77c5249d417b22f510d419d4e/client/post-editor/editor-post-type/index.jsx#L20-L43), a bit similar to what I'm starting to do with [`postTypeSupports` selector](https://github.com/Automattic/wp-calypso/blob/b109436609d955f77c5249d417b22f510d419d4e/client/state/post-types/selectors.js#L41-L69).